blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/PipelineStepImpl.java

```diff
@@ -68,6 +69,10 @@ public String getId() {
 
     @Override
     public String getDisplayName() {
+        String desc = ArgumentsAction.getArgumentDescriptionString(node.getNode());
```

Comment:
Done, UI changes went in logDisplayHelper.js. Probably it needs bit of styling, if we want to show step type as bold or differently to distinguish from description, so maybe ResultItem in JDL needs to change.

![image](https://cloud.githubusercontent.com/assets/38139/25451415/f6e9c1ec-2a76-11e7-846e-3b349ce2197a.png)